### DISCUSSION TITLE
Could you please add Drawable using SVG path?

### DISCUSSION THREAD

**Discussion by daoanhthanh** [dlemstra/magick-wasm#137] [Ideas]
I need an implementation to draw a shape using the SVG path's format. For example, using the command line I can do 
`-draw 'path "M 0,0 L 0,-1853 A 1853,1853 0 0,1  0,1853 Z"'`. I don't know if I can do that using the wasm lib.  If this function is already implemented, kindly tell me how to use it. If not, could you please add it to the next release? 🙏

**Comment by dlemstra** [comment]
That method will become available through an api at a later moment. It will require me to add around 20 classes that will make it possible to draw this but you won't be able to parse the string. You can do that with the `mvg` coder but that is disabled by default on all platforms because of possible security issues. To be able to use that you will need to change the policy and that is also not yet possible in the current api. I am not sure what I will add first, the path api or the policy changes. Another option could be using the browser to draw those lines and the copy the image from the canvas. But I don't know in what kind of environment you would want to use this.

**Reply by daoanhthanh** [reply]
@dlemstra I'm planning to use it in web app

**Reply by dlemstra** [reply]
For your situation it would be best to either use the MVG option that requires a policy change or draw the path on a browser canvas and then load the image from it.
